crypto/rand/rand_lib.c

```diff
@@ -193,6 +193,73 @@ size_t rand_drbg_get_entropy(RAND_DRBG *drbg,
     return ret;
 }
 
+/*
+ * Find a suitable system time.  Start with the highest resolution source
+ * and work down to the slower ones.  This is added as additional data and
+ * isn't counted as randomness, so any result is acceptable.
+ */
+static uint32_t fast_time(void)
```

Comment:
I'm not happy with the name, it sounds a little meaningless to me. I'd rather see it replaced with something like `get_timestamp` or, well, `get_timer_bits`.

Reply:
Yeah, I'm lousy at picking names.